variables.tf

```
@@ -135,11 +135,11 @@ variable "runner_instance" {
 }
 
 variable "runner_ami_filter" {
-  description = "List of maps used to create the AMI filter for the Runner AMI. Must resolve to an Amazon Linux 1 or 2 image."
+  description = "List of maps used to create the AMI filter for the Runner AMI. Must resolve to an Amazon Linux 2023 image."
```

```suggestion
  description = "List of maps used to create the AMI filter for the Runner AMI. Must resolve to an Amazon Linux 1, 2 or 2023 image."
```

suggestion: keep the old Amazon Linux 1 and 2 as your changes are compatible with them. I agree that updating to 2023 makes sense, but this is done with the default below.